ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -353,3 +356,49 @@ class MyWorkflow(BaseWorkflow):
     assert len(adornments) == 2
     assert adornments[0]["label"] == "Try Node"
     assert adornments[1]["label"] == "Retry Node"
+
+
+def test_serialize_workflow__retry_node_edges():
+    """
+    Tests that edges are correctly serialized for a workflow with a retry-adorned CodeExecutionNode and a regular node.
+    """
+
+    @RetryNode.wrap(max_attempts=3, delay=60)
+    class FirstNode(CodeExecutionNode[BaseState, str]):
+        filepath = "./tests/code.py"
```

Comment:
make this `./fixtures/code.py`, and add that file to this repo with the `'def main() -> str:\n    return "Hello, World!"\n'` string we had.

Then, fix our implementation so that we could properly find this file